### ISSUE TITLE
wlroots-hidpi-git is outdated

### ISSUE THREAD

**Issue by FZDSLR** [archlinuxcn/repo#2696]
### 问题类型 / Type of issues

* 过期软件包 / out-of-date packages

### 受影响的软件包 / Affected packages

* wlroots-hidpi-git
* sway-hidpi-git 
----

**Comment by lilacbot** [comment]
| pkgbase | build history | last build log |
| --- | --- | --- |
| sway-hidpi-git | [build history](https://build.archlinuxcn.org/~imlonghao/#sway-hidpi-git) | [last build log](https://build.archlinuxcn.org/imlonghao-api/pkg/sway-hidpi-git/log/1647612564) |
| wlroots-hidpi-git | [build history](https://build.archlinuxcn.org/~imlonghao/#wlroots-hidpi-git) | [last build log](https://build.archlinuxcn.org/imlonghao-api/pkg/wlroots-hidpi-git/log/1647612564) |

**Comment by PeterCxy** [comment]
Could you explain a bit more why you think they are outdated? Both packages have been updated in the past 24 hours.

**Comment by FZDSLR** [comment]
The pkgver of [wlroots-hidpi-git in aur](https://aur.archlinux.org/packages/wlroots-hidpi-git) （0.16.0.r5339.2c59435e-1）is newer than ArchCN （0.15.0.r133.gaaf787ee5-1） and it caused some problems after I add archlinuxCN repository to my repository list.
P.S. But I just noticed [this issue](https://github.com/archlinuxcn/repo/issues/2688) and I guess this is the answer. So I tried to use `sway-hidpi-git` & `wlroots-hidpi-git` in archlinuxCN repository again and didn't meet any problem.
